app/assets/stylesheets/pages/settings.scss

```diff
@@ -92,4 +92,16 @@ $sidebar-item-highlight: #00c081;
     margin-top: 0;
     margin-bottom: 15px;
   }
-}
\ No newline at end of file
+}
+
+.ldap-conn-message-default {
```

Comment:
I guess we don't need this since it will inherit from body and no other class will override it. Remember to remove the js references for this.

Reply:
#ldap_conn_message inherits from h4 which is set to $brand-success color.  I went with h4 on this message for the sake of consistency with the rest of the page.  Thus I kept ldap-conn-message-default style in place so that the message starts out black instead of green ($brand-success).